src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -65,6 +65,11 @@ java.launcher.opt.footer = \
 \    --enable-native-access <module name>[,<module name>...]\n\
 \                  allow code in modules to access code and data outside the Java runtime.\n\
 \                  <module name> can also be ALL-UNNAMED to indicate code on the class path.\n\
+\    --illegal-native-access=<value>\n\
+\                  allow or deny access to code and data outside the Java runtime\n\
+\                  by code in modules for which native access is not explicitly enabled.\n\
+\                  <value> is one of "deny", "warn" or "allow".\n\
+\                  This option will be removed in a future release.\n\
```

Comment:
Should this specify the current default value for this option if it isn't set?

Reply:
We already do, see
https://github.com/openjdk/jdk/pull/19213/files/1c45e5d56c429205ab8185481bc1044a86ab3bc6#diff-d05029afe6aed86f860a10901114402f1f6af4fe1e4b46d883141ab1d2a527b8R582

Reply:
This is slightly different from what we do in the other PR for unsafe memory access where we specify the default in the launcher's help text too https://github.com/openjdk/jdk/pull/19174/files#diff-799093930b698e97b23ead98c6496261af1e2e33ec7aa9261584870cbee8a5eaR219.

I don't have a strong opinion on this, I think either one is fine.

Reply:
Ah, apologies, I was looking in the wrong place. I agree that we should specify default in the launcher, as well as in the man pages.